cmd/juju/resource/list_charm_resources_test.go

```diff
@@ -40,8 +40,9 @@ func (s *ListCharmSuite) TestInfo(c *gc.C) {
 	c.Check(info, jc.DeepEquals, &jujucmd.Info{
 		Name:    "resources",
 		Args:    "<charm>",
-		Purpose: "Display the resources for a charm in the charm store.",
-		Doc: `
+		Purpose: "DEPRECATED: Display the resources for a charm in the charm store.",
+		Doc: `This command is DEPRECATED from Juju 2.3.x
```

Comment:
Replace "from" with "since"? 

It might be good if the help also directed people to the correct command.